ext/spl/tests/SplFileObject_getCurrentLine_invalid_override.phpt

```diff
@@ -4,7 +4,7 @@ Invalid SplFileObject::getCurrentLine() return type
 <?php
 
 class MySplFileObject extends SplFileObject {
-    public function getCurrentLine(): array {
+    public function getCurrentLine(): string {
```

Comment:
This test should keep the wrong type with `#[ReturnTypeWillChange]`, otherwise it no longer tests the code it's supposed to.

Reply:
I've just addressed this comment with my last commit. Additionally, I tried to improve the error message a little bit.